docs: runtime docs + fix runtime missing field (#22816) (#24429)
cosmos/cosmos-sdk: docs/docs/build/building-apps/02-app-mempool.md

* [Sender Nonce Mempool](#sender-nonce-mempool)
* [Priority Nonce Mempool](#priority-nonce-mempool)

> By default, the SDK mempool is a [No-op Mempool](#no-op-mempool), but it can be replaced by the application developer in [`app.go`](./01-app-go-di.md):

Reviewer: ```suggestion
By default, the SDK uses the [No-op Mempool](#no-op-mempool), but it can be replaced by the application developer in [`app.go`](./01-app-go-di.md):
```
